server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java

```diff
@@ -959,6 +961,11 @@ private void addRowRecordForShowQuery(
 
   @Override
   public void delete(DeletePlan deletePlan) throws QueryProcessException {
+    AUDIT_LOGGER.info(
+        "delete data from {} in [{},{}]",
+        deletePlan.getPaths(),
+        deletePlan.getDeleteStartTime(),
+        deletePlan.getDeleteEndTime());
```

Comment:
Same as above.
getPaths returns a list. Can the logger print the paths in this list directly?

Reply:
it can print the array elements directly as follows
![image](https://user-images.githubusercontent.com/67779101/124537523-a14d2380-de4c-11eb-9a00-78b8938f912f.png)

Reply:
it can print  a list as follows
![image](https://user-images.githubusercontent.com/67779101/124538281-efaef200-de4d-11eb-96d3-d84f52ee0eed.png)